canonical-websites/partners.ubuntu.com
# Add charm partner program page

# Details
- Card: https://canonical.leankit.com/Boards/View/111185042/113934420
## Done
- Created the CPP page on programmes
- Wired it up in views.py
- Create a block template for the logo lists
### QA
- Go to /programmes/charm and check it looks ok
- Check the logos list row does now appear as there is no logos
